PeepPeep/FeedBack/TotalActivityReport.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  TotalActivityReport.swift
+//  FeedBack
+//
+//  Created by Ha Jong Myeong on 2023/07/12.
+//
+
+import DeviceActivity
+import ManagedSettings
+import SwiftUI
+
+extension DeviceActivityReport.Context {
+    static let totalActivity = Self("Total Activity")
+}
+
+struct TotalActivityReport: DeviceActivityReportScene {
+    let context: DeviceActivityReport.Context = .totalActivity
+    let content: (ActivityReport) -> TotalActivityView // ActivityReport를 입력으로 받아 TotalActivityView를 반환하는 클로저
```

Comment:
```suggestion
     /// ActivityReport를 입력으로 받아 TotalActivityView를 반환하는 클로저
    let content: (ActivityReport) -> TotalActivityView
```
/가 3개면 자동완성에서 설명으로 뜹니당 ! : )